src/simpla-text-editor.html

```diff
@@ -107,9 +205,10 @@
         this.is = 'simpla-text-toolbar';
 
         this.properties = {
+
           tools: {
             type: Object,
-            value: INITIAL_TOOLS
+            value: () => INITIAL_TOOLS
```

Comment:
There's a subtle difference, so if it's being stored in a constant somewhere outside, and you return that variable rather than an object literal, it'll share the same object. So you need to have a function which returns a new object literal each time - so `INITIAL_TOOLS` was a function to return an object literal. It was a bad name - should really be `getInitialTools` or something. But yeah - if I've done it before with objects then that was wrong ha.

I wish Polymer just enforced `value` to be a function, rather than allowing it to be either a value or a function, gets confusing.